code/game/machinery/kitchen/microwave.dm

```diff
@@ -269,10 +254,6 @@
 				if (istype(O,/obj/item/reagent_containers/food/snacks/meat)) //any meat
 					items_measures[display_name] = "slab of meat"
 					items_measures_p[display_name] = "slabs of meat"
-				if (istype(O,/obj/item/reagent_containers/food/snacks/donkpocket))
```

Comment:
It feels like microwaves should still be able to cook donk/sinpockets, considering their inspiration is basically designed to be microwavable.

Reply:
They're still capable of microwaving donk-pockets. I am legally disallowed from shipping a microwave that cannot heat hot pockets. They just display their actual names now; instead of "3 turnovers" you'll see "3 warm Donk-pockets", for example.